src/community/components/tag/tag.module.scss

```
@@ -64,9 +64,9 @@
   min-height: 1.5rem;
   padding: 0.125rem 0.5rem;
   font-size: 0.875rem;
-  vertical-align: bottom; // Removes extra pixel added to height by inline-flex
   text-align: center;
   text-decoration: none;
+  vertical-align: bottom; // Removes extra pixel added to height by inline-flex
```

Stylelint (17.13.0)

[error] Expected "vertical-align" to come before "text-decoration" (order/properties-order)

(order/properties-order)